chain/jsonrpc/res/debug.html

```diff
@@ -24,9 +24,16 @@
                 type: "GET",
                 url: "/status",
                 success: data => {
+                    let binaryText = $('<span>')
+                        .text(`version "${data.version.version}" built with rustc "${data.version.rustc_version}" at `);
```

Comment:
```suggestion
                        .text(`version ${data.version.version} built with rustc ${data.version.rustc_version} from `);
```

Reply:
I'm preferring accuracy to a pretty UI here... I originally also did it without the quotes but then everything is in the same font and it's less clear what the exact information is. Was thinking of also <span style=...>ing all these fields but that makes the code messy because we just have jQuery here.

Reply:
All of those are single word strings. IMO they don’t need any special styling or quoting. But again, I don’t have strong opinions.